### PR TITLE
Update course content to remove `gh pr create`

### PR DESCRIPTION
**Pull request by heiskr**
Update course content to remove `gh create pr`

@cmwilson21 if you can focus on the README, and @hectorsector on the actions 🙏🏼 

Resolves https://github.com/skills/review-pull-requests/issues/28

### CODE REVIEW

File: .github/workflows/2-assign-yourself.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
       # Update README to close <details id=2> and open <details id=3>
       # and set STEP to '3'
       - name: Update to step 3
-        uses: skills/action-update-step@v1
+        uses: skills/action-update-step@v2
```
Comment (heiskr) [inline .github/workflows/2-assign-yourself.yml:63]:
```suggestion
        uses: skills/action-update-step@v1
```

File: .github/workflows/4-suggest-changes.yml [hunk 2]
```diff
@@ -52,7 +52,7 @@ jobs:
     steps:
       # We'll need to check out the repository so that we can edit the README
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```
Comment (heiskr) [inline .github/workflows/4-suggest-changes.yml:55]:
```suggestion
        uses: actions/checkout@v3
```

File: .github/workflows/4-suggest-changes.yml [hunk 3]
```diff
@@ -26,15 +26,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```
Comment (heiskr) [inline .github/workflows/4-suggest-changes.yml:29]:
```suggestion
        uses: actions/checkout@v3
```

**Review by chadfawcett — APPROVED** [review]

**Comment by heiskr** [comment]
Given the urgency, I'm going to go ahead and merge, but @skills/content-reviewers feel free to make edits as needed.

**Pull request merged**